docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,263 @@
+protocol > otlp > retry | [retry][] | Configures retry mechanism for failed requests. | no
+
+The `>` symbol indicates deeper levels of nesting. For example, `resolver > static`
+refers to a `static` block defined inside a `resolver` block.
+
+[resolver]: #resolver-block
+[static]: #static-block
+[dns]: #dns-block
+[protocol]: #protocol-block
+[otlp]: #otlp-block
+[client]: #client-block
+[tls]: #tls-block
+[keepalive]: #keepalive-block
+[queue]: #queue-block
+[retry]: #retry-block
+
+### resolver block
+
+The `resolver` block configures how to retrieve the endpoint to which this exporter will send data.
+
+Inside the `resolver` block, either the [dns][] block or the [static][] block 
+should be specified. If both `dns` and `static` are specified, `dns` takes precedence.
+
+### static block
+
+The `static` block configures a list of endpoints which this exporter will send data to.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`hostnames` | `list(string)` | List of endpoints to export to. |  | yes
+
+### dns block
+
+The `dns` block periodically resolves an IP address via the DNS `hostname` attribute. This IP address 
+and the port specified via the `port` attribute will then be used by the gRPC exporter 
+as the endpoint to which to send data to.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`hostname` | `string`   | DNS hostname to resolve. |  | yes
+`port`     | `string`   | Port to be used with the resolved IP address. | `"4317"` | no
+`interval` | `duration` | Resolver interval. | `"5s"` | no
+`timeout`  | `duration` | Resolver timeout. | `"1s"`  | no
+
+### protocol block
+
+The `protocol` block configures protocol-related settings for exporting.
+At the moment only the OTLP protocol is supported.
+
+### otlp block
+
+The `otlp` block configures OTLP-related settings for exporting.
+
+### client block
+
+The `client` block configures the gRPC client used by the component. 
+The endpoints used by the client block are the ones from the `resolver` block
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`compression` | `string` | Compression mechanism to use for requests. | `"gzip"` | no
+`read_buffer_size` | `string` | Size of the read buffer the gRPC client to use for reading server responses. | | no
+`write_buffer_size` | `string` | Size of the write buffer the gRPC client to use for writing requests. | `"512KiB"` | no
+`wait_for_ready` | `boolean` | Waits for gRPC connection to be in the `READY` state before sending data. | `false` | no
+`headers` | `map(string)` | Additional headers to send with the request. | `{}` | no
+`balancer_name` | `string` | Which gRPC client-side load balancer to use for requests. | | no
```

Comment:
Isn't the default here `pick_first`?

Reply:
This still feels a bit odd, the functional default is `pick_first` . @clayton-cornell for his thoughts on it. 
`When unspecified, `pick_first` is used.` -> This feels like the definition of a default when applied to a single field.